### PR TITLE
how-to: refine language

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? 

This PR refines the language in `how-to` documents.

### What is the related PR or file link(s)? 

https://github.com/pingcap/docs-cn/pull/1844
https://github.com/pingcap/docs-cn/pull/1899
https://github.com/pingcap/docs-cn/pull/1880
https://github.com/pingcap/docs/pull/1597

### Which version does your change affect? 

`dev`, `v2.1`, `v3.0`

### CODE REVIEW

File: dev/how-to/deploy/data-migration-with-binary.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ Usage of worker:
   -checker-backoff-max duration
         任务检查模块中，检查出错误后等待自动恢复的最长时间间隔（默认值："5m0s"，一般情况下不需要修改。如果对该参数的作用没有深入的了解，不建议修改该参数）
   -checker-backoff-rollback duration
-        任务检查模块中，定时调整恢复等待时间的间隔（默认值："5m0s"，一般情况下不需要修改，如果对该参数的作用没有深入的了解，不建议修改该参数）
+        任务检查模块中，自动恢复等待时间的调整间隔（默认值："5m0s"，一般情况下不需要修改，如果对该参数的作用没有深入的了解，不建议修改该参数）
```
Comment (lilin90) [inline dev/how-to/deploy/data-migration-with-binary.md]: How to understand `调整间隔`?

File: dev/how-to/maintain/tidb-binlog.md [hunk 2]
```diff
@@ -332,7 +332,7 @@ Pump 以 paused 状态退出进程时，不保证所有 binlog 数据被下游 D
 
 1. 查看 drainer.log 日志，查找 `exec failed` 找到 Drainer 退出前最后一条执行失败的 DDL。
 2. 将 DDL 改为下游兼容支持的版本，在下游数据库中手动执行。
-3. 查看 drainer.log 日志，查找 `执行失败的DDL语句`，可以查询到该 DDL 的 commit-ts。
+3. 查看 drainer.log 日志，查找执行失败的DDL语句，可以查询到该 DDL 的 commit-ts。
```
Comment (lilin90) [inline dev/how-to/maintain/tidb-binlog.md]:
```suggestion
3. 查看 drainer.log 日志，查找执行失败的 DDL 语句，可以查询到该 DDL 的 commit-ts。
```

File: dev/how-to/deploy/data-migration-with-binary.md (hunk 1 above)
Comment (TomShawn) [reply dev/how-to/deploy/data-migration-with-binary.md]: 调整自动恢复等待时间的间隔

**Comment by lilin90** [comment]
@WangXiangUSTC PTAL

File: dev/how-to/deploy/data-migration-with-binary.md [hunk 3]
```diff
@@ -83,6 +85,14 @@ Usage of worker:
         DM-worker 的地址
 ```
 
+> **注意：**
+>
+> 某些情况下，无法使用命令行参数的方法来配置 DM-worker，因为有的配置并未暴露给命令行。
+
+**配置方式 2：配置文件**
+
+推荐使用配置文件来配置 DM-worker，把以上配置内容写入到 `conf/dm-worker1.toml` 中。
```
Comment (WangXiangUSTC) [inline dev/how-to/deploy/data-migration-with-binary.md]: 不能写`以上配置内容`吧，因为命令行参数和配置文件里的参数的名称可能是不一样的，而且配置文件中的配置项一般会更多一点

File: dev/how-to/deploy/data-migration-with-binary.md [hunk 4]
```diff
@@ -172,6 +184,14 @@ Usage of dm-master:
         打印出 DM-master 的示例配置
 ```
 
+> **注意：**
+>
+> 某些情况下，无法使用命令行参数的方法来配置 DM-worker，因为有的配置并未暴露给命令行。
+
+**配置方式 2：配置文件**
+
+推荐使用配置文件，把以上配置内容写入到 `conf/dm-master.toml` 中。
```
Comment (WangXiangUSTC) [inline dev/how-to/deploy/data-migration-with-binary.md]: 同上

**Comment by WangXiangUSTC** [comment]
rest LGTM

**Comment by lilin90** [comment]
@TomShawn Please resolve new comments~

**Comment by TomShawn** [comment]
@WangXiangUSTC PTAL again, thanks.

**Comment by WangXiangUSTC** [comment]
LGTM

**Pull request merged**